services/account_test.go

```diff
@@ -0,0 +1,137 @@
+package services
+
```

Comment:
Do we make separate test files for each service or one big test like Igor advised us yesterday?